pyproject.toml

```diff
@@ -33,15 +33,17 @@ python = "^3.10"
 tenacity = "^8.4.2"
 pure-sasl = "^0.6.2"
 cosl = "^0.0.12"
-ops = "^2.13.0"
+pydantic = "^1.10.17"
+pyyaml = "^6.0.1"
+poetry-plugin-export = "^1.8.0"
 
 # TODO: clean any of the notes below and their deps.
 [tool.poetry.group.charm-libs.dependencies]
 # data_platform_libs/v0/data_interfaces.py
 ops = "^2.13.0"
 poetry-core = "^1.9.0"
 # tls_certificates_interface/v3/tls_certificates.py
-cryptography = "*"
+cryptography = "42.0.8"
```

Comment:
Pinning it down to avoid
![image](https://github.com/user-attachments/assets/de7648b4-76aa-40ce-b3dc-b438933fdfbb)

Providing a fix in https://github.com/canonical/opensearch-dashboards-operator/pull/82